conf/evolutions/default/6.sql

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2016 LinkedIn Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# --- !Ups
+
+/**
+ * This query make neccesssary steps to run IPSO
+ */
+
+ALTER TABLE tuning_algorithm MODIFY COLUMN optimization_algo enum('PSO','PSO_IPSO') NOT NULL COMMENT 'optimization algorithm name e.g. PSO' ;
+
+INSERT INTO tuning_algorithm VALUES (3, 'PIG', 'PSO_IPSO', '3', 'RESOURCE', current_timestamp(0), current_timestamp(0));
+
+
+INSERT INTO tuning_parameter VALUES (10,'mapreduce.task.io.sort.mb',3,100,50,1920,50, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (11,'mapreduce.map.memory.mb',3,2048,1024,8192,1024, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (12,'mapreduce.task.io.sort.factor',3,10,10,150,10 ,0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (13,'mapreduce.map.sort.spill.percent',3,0.8,0.6,0.9,0.1, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (14,'mapreduce.reduce.memory.mb',3,2048,1024,8192,1024, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (15,'pig.maxCombinedSplitSize',3,536870912,536870912,536870912,128, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (16,'mapreduce.reduce.java.opts',3,1536,500,6144,64, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (17,'mapreduce.map.java.opts',3,1536,500,6144,64, 0, current_timestamp(0), current_timestamp(0));
+INSERT INTO tuning_parameter VALUES (18,'mapreduce.input.fileinputformat.split.maxsize',3,536870912,536870912,536870912,128, 1, current_timestamp(0), current_timestamp(0));
+
+
+CREATE TABLE IF NOT EXISTS tuning_parameter_constraint (
+  id int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Auto increment unique id',
+  job_definition_id int(10) unsigned NOT NULL COMMENT 'Job Definition ID',
+  tuning_parameter_id int(100)  unsigned NOT NULL COMMENT 'tuning_parameter_id ',
+  constraint_type enum('BOUNDARY','INTERDEPENDENT') NOT NULL COMMENT 'Constraint ID',
+  lower_bound double COMMENT 'Lower bound of parameter',
+  upper_bound double COMMENT 'Upper bound of parameter',
+  param_name varchar(100) NOT NULL COMMENT 'name of the hadoop parameter e.g. mapreduce.task.io.sort.mb ',
```

Comment:
This is needed in IPSO Manager to compare parameter names ,with the one on which IPSO should be applied. We can get this by joining tuning_parameter and tuning_parameter_constraint. But to avoid join , i have inserted param_name , while inserting initial values in tuning parameter constraint